### ISSUE TITLE
I think the solution is elsewhere: treat ALL patents as RANDR by requirement of law.

### ISSUE THREAD

**Issue by BrazilianJoe** [twitter/innovators-patent-agreement#4]
The current patent law was made in a time where one could actually be an inventor just making crazy contraptions in his basement/garage/barn, and was designed to make sure the inventor would be able to profit instead of being ripped off.

In the current times, giant corporations are creating countless patents, and it has been shown time and again that the patent system is in many instances simply unable to verify the 'obviousness' or prior art which would make the patents invalid. 

These treasure troves of patents are also used not only to wage war, but also to stifle innovation and attempt to monopolize the market. 

I think that the solution is not on voluntary agreements, but on a pivotal change in the way the law deals with patents. 
The solution, I believe, is that ALL patents should be RANDR by requirement of law. That would render much of the current block product from market shenanigans obsolete, and the discussion would move on to 'how much should be charged for it'.

I propose an arbitrary upper limit of 10% (could be any other value, but not too high) of the product value when leaving the hands of the manufacturer to be reserved to share amongst patent holders.

It would be the duty of the patent holder to set how much it wants for each patent or package. If all patents involved set the cost to less than the 10%, the better for the manufacturer. If the sum of the licensing costs are more than 10%, the licensing costs for each patent are used as a weight and the 10% are divided proportionally between all patent holders.

When a component (such as a microchip) uses licensed patents and patent exhaustion applies, these patents can still be used in the calculation of patent licensing costs, to reduce the amount to be paid, but the money does not need to exchange hands, since the component (microchip in the example) producer has already paid fully for them. 

It would be the duty of a component seller to inform all patents in use by its product and their status (i.e. if exhaustion or other legalities apply) to the buyer.

A system like that would greatly simplify the matters of the current patent system, and would still protect the patent holders, while also would prevent behemoth patent holders and non-practicing entities i.e. "patent trolls" from abusing their dominant position, the latter being the real problem being faced right now, which this aims to solve.

**Comment by jamesarosen** [comment]
There are many people working on reforming intellectual property law in the US, and I applaud their efforts. Until they're successful, however, a project like this can really help our industry.

**Comment by benltwitter** [comment]
Yes, this proposal should not be seen as remedying/excluding/obviating the need for a myriad of worthy reforms to the law on patents.
